http://bugs.debian.org/79358

this is not a proper fix, but should unbreak quite a few things.

-- 
 hurd/hurdselect.c |    2 ++
 1 file changed, 2 insertions(+)
--- a/hurd/hurdselect.c
+++ b/hurd/hurdselect.c
@@ -84,6 +84,8 @@
 
       to = (timeout->tv_sec * 1000 +
             (timeout->tv_nsec + 999999) / 1000000);
+      if (strcmp(program_invocation_short_name, "vi") && strcmp(program_invocation_short_name, "vim") && strcmp(program_invocation_short_name, "vimdiff") && !to)
+	to = 1;
     }
 
   if (sigmask && __sigprocmask (SIG_SETMASK, sigmask, &oset))
